### PR TITLE
Encode JSON with the Django encoder

### PR DESCRIPTION
**Pull request by plfiorini**
Use DjangoJSONEncoder which supports more data types such as Decimal.

### CODE REVIEW

File: smart_selects/views.py [hunk 1]
```diff
@@ -50,7 +52,7 @@ def filterchain(request, app, model, field, foreign_key_app_name, foreign_key_mo
         sort_results(results)
 
     serialized_results = serialize_results(results)
-    results_json = json.dumps(serialized_results)
+    results_json = json.dumps(serialized_results, cls=DjangoJSONEncoder)
     return HttpResponse(results_json, content_type='application/json')
```
Comment (webknjaz) [inline smart_selects/views.py]: Alternatively, `JsonResponse` could be used.

**Comment by SalahAdDin** [comment]
:+1:

File: smart_selects/views.py (hunk 1 above)
Comment (plfiorini) [inline smart_selects/views.py]: @webknjaz Yep that sounds right, updated

**Comment by SalahAdDin** [comment]
@digi604 Hey!

**Comment by millin** [comment]
@jimmykobe1171, where are you?

**Comment by blag** [comment]
I just became the GitHub and PyPI maintainer - I'll be merging this later tonight. Thanks for your help!

**Comment by plfiorini** [comment]
@blag Wonderful thank you very much :D

**Pull request merged**